response_test.go

```diff
@@ -22,10 +24,11 @@ func ExamplePreloadableResponse() {
 /******** Response tests. ********/
 
 var responseTests = []struct {
-	function       func(resp *http.Response) (header *string, issues Issues)
+	function       func(resp *http.Response, policy preloadlist.PolicyType) (header *string, issues Issues)
 	description    string
 	hstsHeaders    []string
 	expectedIssues Issues
+	policy         preloadlist.PolicyType
```

Comment:
Don't change this type here. The policy is irrelevant for most of the tests.

Since EligibleResponse needs a policy parameter passed in, you can work around that with a wrapper function. E.g. if you want to test EligibleResponse with a policy of preloadlist.Bulk1Year, create an EligibleResponseBulk1Year function that only takes in a domain, and calls EligibleResponse with the passed in domain and a fixed policy of Bulk1Year.

Once those changes are made, you can remove the wrapper functions in header.go and response.go, and only the EligibleResponse* functions take in a policy.

Reply:
I removed the Wrapper functions in response.go but not in header.go because those are used in checkResponse() in response.go